src/main/java/nextstep/courses/domain/image/ImageSize.java

```diff
@@ -0,0 +1,49 @@
+package nextstep.courses.domain.image;
+
+public class ImageSize {
+    private static final int MIN_WIDTH = 300;
+    private static final int MIN_HEIGHT = 200;
+    private static final int MAX_VOLUME = 1_048_576;
```

Comment:
일반적으로 용량을 나타낼 때는 바이트 단위 수식으로 나타냅니다.
1MB의경우 아래처럼 표현하면 쉽고 빠르게 파악 가능합니다.
```suggestion
    private static final int MAX_VOLUME = 1 * 1024 * 1024;
```

Reply:
처음알았네요.. 감사합니다